### DISCUSSION TITLE
GitHub Packages, Support for Linux packages

### DISCUSSION THREAD

**Discussion by nimarty** [community/community#36023] [Packages]
I'd like to publish Linux ipk packages on GitHub. These packages work together with an OPKG package server. This is a simple HTTP server which provides a Packages.gz file with meta information and the ipk package files.

Maybe there could be some kind of "generic" package type which can be configured. It should allow for public anonymous access.

**Comment by Matthias-Mangin** [comment]
@nimarty have you found a way to do this ? I have a similar issue where I'd like to upload a Data.zip and use it in my Github Actions jobs. It works on gitlab with generic packages, but it seems this feature isn't yet implemented on github :disappointed:  https://github.com/github/roadmap/issues/119

**Reply by nimarty** [reply]
I wrapped my ipk packages in a docker image and pushed that to ghcr.io
https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry

**Comment by johnc-ftl** [comment]
I posted #38083 - related, but more generalized.
